frame/base-fee/src/tests.rs

```diff
@@ -35,9 +35,9 @@ use crate::BaseFeeThreshold as BaseFeeThresholdT;
 
 parameter_types! {
 	pub const BlockHashCount: u64 = 250;
-	pub BlockWeights: frame_system::limits::BlockWeights =
-		frame_system::limits::BlockWeights::simple_max(Weight::from_parts(1024, 0));
 }
+
+#[derive_impl(frame_system::config_preludes::TestDefaultConfig as frame_system::DefaultConfig)]
```

Comment:
It seems that enabling this can simplify configs in the frame_system::Config.

Reply:
Yeah, you could simplify the system config and use runtime v2 syntax in later PRs

Reply:
BTW, you'd better not to merge the dependabot PRs when there is a deps upgrade PR

Reply:
Okay. Let's do it later in seperated pr.